http/legacy/ping_handler.go

```diff
@@ -24,7 +26,7 @@ func NewPingHandler(version string) *PingHandler {
 
 // handlePostLegacyWrite is the HTTP handler for the POST /write route.
 func (h *PingHandler) pingHandler(w http.ResponseWriter, r *http.Request) {
-	w.Header().Add("X-Influxdb-Build", "oss2")
-	w.Header().Add("X-Influxdb-Version", h.InfluxDBVersion)
+	w.Header().Set("X-Influxdb-Build", h.InfluxDBBuildType)
```

Comment:
The top-level `buildHeader` middleware in `http/handler.go` is already taking care of setting this header information for all requests, which is why it was duplicated here. So instead of setting the header info again, I'd prefer it if we simply removed the `w.Header()...` lines from `pingHandler` since at this point they aren't doing anything. Then you won't need the `buildType` constant to be threaded through to here.

Reply:
Fixed